src/App.js

```diff
@@ -18,30 +19,147 @@ function App() {
 
   return (
     <>
-      <Navbar />
-      <Routes>
-        <Route
-          path="/"
-          element={isLoggedIn ? <Home /> : <Navigate to="/signIn" />}
-        />
-        <Route path="/register" element={<UserRegistration />} />
-        <Route path="/signIn" element={<SignIn />} />
-        <Route path="/home" element={<Home />} />
-        <Route path="/checkout" element={<CheckoutPage />} />
-        <Route path="/wishlist" element={<WishListView />} />
-        <Route path="/checkout/completed" element={<CheckoutComplete />} />
-        <Route path="/profile" element={<UserProfile />} />
-        <Route
-          path="/chat"
-          element={
-            <AlbumChat spotifyUrl="https://api.spotify.com/v1/albums/6r1lh7fHMB499vGKtIyJLy" />
-          }
-        />
-        {/* Below route is the catch-all route. It **MUST** be the last route because react-router checks the routes from top-to-bottom */}
-        <Route path="*" element={<PageNotFound />} />
-      </Routes>
+      <AuthProvider>
```

Comment:
Let's keep all the provider elements (like BrowserRouter which provides us with access to react-router hooks, and AuthProvider which provides us with access to auth hooks) in the `index.js` file; just for clearer separation of responsibiliites.

Reply:
Hope, I did right😅